code/game/machinery/vending.dm

```diff
@@ -1118,17 +1118,30 @@ var/global/num_vending_terminals = 1
 	name = "PTech"
 	desc = "Cartridges for PDAs"
 	product_slogans = "Carts to go!"
+	req_access_txt = "15"
```

Comment:
req_access_txt is reserved for mapping. You should set req_access directly in code, and that should be done using the access defines, not magic numbers.